Vagrantfile

```diff
@@ -22,6 +22,10 @@ Vagrant.configure(2) do |config|
         master.vm.network "forwarded_port", guest: 8080, host: 8081
         master.vm.provision "shell", inline: $base_script
         master.vm.provision "shell", inline: $script
+
+        master.vm.provider "virtualbox" do |provider|
+            provider.memory = 640
```

Comment:
Why does this become necessary? This change should probably be split out into its own commit, providing a proper reasoning in the commit message.